server/src/models/mentors.model.js

```diff
@@ -1,27 +1,32 @@
 import mongoose from 'mongoose';
 
-const mentorsSchema = {
+const mentorsSchema = new mongoose.Schema({
     mentor_name: {
-        type: String, required: true
+        type: String
```

Comment:
You can probably add the required props back here as we identified the real issue :) I think it is good to have the additional check in your schema to ensure that only valid data will be saved to the database.